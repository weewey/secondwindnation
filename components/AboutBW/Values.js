import React from 'react';
import Typography from 'material-ui/Typography';
import { withStyles } from 'material-ui/styles';
import PropTypes from 'prop-types';
import Grid from 'material-ui/Grid';
import Hidden from 'material-ui/Hidden';


const styleSheet = theme => ({
  root: {
    width: '100%',
    textAlign: 'center',
    // backgroundColor: 'yellow',
    justifyContent: 'center',
    display: 'flex',
    alignItems: 'center',
  },
  para: {
    maxWidth: 1200,
    padding: 20,
    margin: 'auto',
  },
  grid: {
    maxWidth: '40em',
  },
  title: {
    // fontWeight: 500,
    color: 'black',
  },
  text: {
    fontWeight: 100,
  }
});

function Values(props) {
  const classes = props.classes;
  return (
    <div className={classes.root}>
      <div className={classes.para}>
        <Grid container spacing={40} align="center" justify="center">
          <Grid item xs={12}>
            <Typography type="display3" className={classes.title}>VALUES</Typography>
          </Grid>
          <Grid item xs={12} sm={12} md={4} className={classes.grid}>
            <Typography type="title">COMPASSION</Typography>
            <Typography type="subheading" className={classes.text}>We believe practicing compassion in everything we do enables us to help individuals find their Second Wind, by creating an environment where everyone is welcomed to explore, innovate and breakthrough.</Typography>
          </Grid>
          <Grid item xs={12} sm={12} md={4} className={classes.grid}>
            <Typography type="title">BETTERMENT</Typography>
            <Typography type="title">OF SELF</Typography>
            <Typography type="subheading" className={classes.text}>We believe in seeking self-betterment everything we do. By pursuing the betterment of self, Second Wind Nation believes every individual can achieve their personal goals.</Typography>
          </Grid>
          <Grid item xs={12} sm={12} md={4} className={classes.grid}>
            <Typography type="title">BEING</Typography>
            <Typography type="title">COMMUNITY</Typography>
            <Typography type="title">ORIENTED</Typography>
            <Typography type="subheading" className={classes.text}>We believe in leveraging on each otherâ€˜s strengths and harnessing the collective energy of the swim bike run community. Second Wind Nation exists to serve this community better everyday.</Typography>
          </Grid>
        </Grid>
      </div>
    </div>
  );
}


Values.propTypes = {
  classes: PropTypes.object.isRequired,
};

export default withStyles(styleSheet)(Values);
