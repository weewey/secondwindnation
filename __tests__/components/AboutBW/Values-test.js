import React from 'react';
import renderer from 'react-test-renderer';

import Values from '../../../components/AboutBW/Values';

describe('Values component', () => {
  it('should render', () => {
    const value = renderer.create(<Values />,);
    const tree = value.toJSON();
    expect(tree).toMatchSnapshot();
  },
);
})
