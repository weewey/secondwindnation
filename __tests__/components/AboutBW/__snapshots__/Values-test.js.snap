// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Values component should render 1`] = `
<div
  className="Values-root-1"
>
  <div
    className="Values-para-2"
  >
    <div
      align="center"
      className="MuiGrid-typeContainer-6 MuiGrid-spacing-xs-40-28 MuiGrid-justify-xs-center-21"
    >
      <div
        className="MuiGrid-typeItem-7 MuiGrid-grid-xs-12-41"
      >
        <h1
          className="MuiTypography-root-94 MuiTypography-display3-96 Values-title-4"
        >
          VALUES
        </h1>
      </div>
      <div
        className="MuiGrid-typeItem-7 MuiGrid-grid-xs-12-41 MuiGrid-grid-sm-12-54 MuiGrid-grid-md-4-59 Values-grid-3"
      >
        <h2
          className="MuiTypography-root-94 MuiTypography-title-100"
        >
          COMPASSION
        </h2>
        <h3
          className="MuiTypography-root-94 MuiTypography-subheading-101 Values-text-5"
        >
          We believe practicing compassion in everything we do enables us to help individuals find their Second Wind, by creating an environment where everyone is welcomed to explore, innovate and breakthrough.
        </h3>
      </div>
      <div
        className="MuiGrid-typeItem-7 MuiGrid-grid-xs-12-41 MuiGrid-grid-sm-12-54 MuiGrid-grid-md-4-59 Values-grid-3"
      >
        <h2
          className="MuiTypography-root-94 MuiTypography-title-100"
        >
          BETTERMENT
        </h2>
        <h2
          className="MuiTypography-root-94 MuiTypography-title-100"
        >
          OF SELF
        </h2>
        <h3
          className="MuiTypography-root-94 MuiTypography-subheading-101 Values-text-5"
        >
          We believe in seeking self-betterment everything we do. By pursuing the betterment of self, Second Wind Nation believes every individual can achieve their personal goals.
        </h3>
      </div>
      <div
        className="MuiGrid-typeItem-7 MuiGrid-grid-xs-12-41 MuiGrid-grid-sm-12-54 MuiGrid-grid-md-4-59 Values-grid-3"
      >
        <h2
          className="MuiTypography-root-94 MuiTypography-title-100"
        >
          BEING
        </h2>
        <h2
          className="MuiTypography-root-94 MuiTypography-title-100"
        >
          COMMUNITY
        </h2>
        <h2
          className="MuiTypography-root-94 MuiTypography-title-100"
        >
          ORIENTED
        </h2>
        <h3
          className="MuiTypography-root-94 MuiTypography-subheading-101 Values-text-5"
        >
          We believe in leveraging on each otherâ€˜s strengths and harnessing the collective energy of the swim bike run community. Second Wind Nation exists to serve this community better everyday.
        </h3>
      </div>
    </div>
  </div>
</div>
`;
